Sources/SwiftRefactor/OpaqueParameterToGeneric.swift

```diff
@@ -0,0 +1,213 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftSyntax
+
+fileprivate typealias RewrittenSome = (
```

Comment:
Any reason not to just make this a struct or at least add names? Is `ConstrainedSugarTypeSyntax` used anywhere?